analysis/viewshed-geoelement/README.md

```diff
@@ -1,29 +1,39 @@
-# Viewshed GeoElement
+# Viewshed geoelement
 
-Attach a viewshed to an object to visualize what it sees.
+Analyze the viewshed for an object (GeoElement) in a scene.
 
-![](ViewshedGeoElement.gif)
+![Image of viewshed for geoelement](ViewshedGeoElement.png)
+
+## Use case
+
+A viewshed analysis is a type of visual analysis you can perform on a scene. The viewshed aims to answer the question 'What can I see from a given location?'. The output is an overlay with two different colors - one representing the visible areas (green) and the other representing the obstructed areas (red).
 
 ## How to use the sample
 
-Once the scene is done loading, click on a location for the tank to drive to. It will automatically turn and drive straight towards the clicked point. The viewshed will automatically move and rotate with the tank.
+Click to set a destination for the vehicle (a GeoElement). The vehicle will 'drive' towards the tapped location. The viewshed analysis will update as the vehicle moves.
 
 ## How it works
 
-To attach a viewshed to a `GeoElement`:
+1. Create and show the scene, with an elevation source and a buildings layer.
+2. Add a model (the `GeoElement`) to represent the observer (in this case, a tank).
+    * Use a `SimpleRenderer` which has a heading expression set in the `GraphicsOverlay`. This way you can relate the viewshed's heading to the `GeoElement` object's heading.
+3. Create a `GeoElementViewshed` with configuration for the viewshed analysis.
+4. Add the viewshed to an `AnalysisOverlay` and add the overlay to the scene.
+5. Configure the SceneView `CameraController` to orbit the vehicle.
```

Comment:
`CameraController` is a parent class in our API, we should probably be more specific here with `OrbitGeoElementCameraController`

Reply:
Thanks, updated!